_config.yml

```diff
@@ -0,0 +1,26 @@
+remote_theme: just-the-docs/just-the-docs
+plugins:
+  - jekyll-remote-theme
+  - jekyll-relative-links
+relative_links:
+  enabled: true
+  collections: true
+include:
+  - README.md
+  - _licenses.md
+exclude:
+  - testing/data/DATA_LICENSES.md
+title: F3D
+description: Fast and minimalist 3D viewer
+logo: /resources/logotype.svg
+color_scheme: dark
+
+# Aux links for the upper right navigation
+aux_links:
+  "<img src=\"https://user-images.githubusercontent.com/3129530/197903476-11d9c41d-73b0-461f-93e8-c8c00c680e6e.png\" width=\"30px\">":
+    - "//github.com/f3d-app/f3d"
```

Comment:
> The link "F3D on github" on the top right isn't great. Can you maybe add a github logo and a better text like "Explore code source on GitHub"

Turns out, we can, and it looks great ! @Meakk

Reply:
Cool!